portfolio/src/main/java/com/google/sps/Puzzle.java

```diff
@@ -13,6 +13,11 @@
    */
   private String imageUrl;
 
+  /*
+  * Difficulty of the puzzle
+  */
+  private String difficulty;
```

Comment:
Use in enum instead of a String. It will ensure that the value is one of the valid enum types. You can go from a string to an enum using [Enum.valueOf](https://www.tutorialspoint.com/java/lang/enum_valueof.htm).